### ISSUE TITLE
MathML is ignored by NVDA by default

### ISSUE THREAD

**Issue by G0dwin** [nvaccess/nvda#17667]
NVDA doesn't support MathML out of the box, it requires additional components to read, other screen readers support Math out of the box. While that's not ideal, there may be good reasons for this. However, when NVDA reaches a `<math>` tag in the browser, it ignores the content completely, not allowing the user to know that there is an expressions there at all

If supporting MathML out of the box is challenging, NVDA needs to at least announce that there is content which it cannot read. Otherwise users will not be aware that the content is even there in the first place. As a simple fallback, it should at least read the role, something like "math expression"

### Steps to reproduce:

1. Install NVDA
2. Do not install any additional plugins or components
3. Navigate to this URL: https://support.freedomscientific.com/SurfsUp/MathML-Samples.html

### Actual behavior:

```text
MathML Examples
heading    level 2  Area of a circle

heading    level 2  Quadratic Equation

heading    level 2  Quadratic Formula

heading    level 2  Embedded table

heading    level 2  Semantics (MathML tag - with content markup annotation)


heading    level 2  Polynomial Equation (with mstyle)
a x^2+b x+c=0  
heading    level 2  Complex Equation

heading    level 2  E Equals MC Squared in Table

heading    level 2  Broken Fences

heading    level 2  Fenced 1

heading    level 2  GroupAligned


heading    level 2  Sine function

heading    level 2  Enclosed

heading    level 2  Cubed Root of x

heading    level 2  Fourth Root of 64

heading    level 2  Cubed Root of x +1


heading    level 2  Fifth Root of x

heading    level 2  Minus one third Root of the quanity x plus 1
```

### Expected behavior:

After each of these headings, we should hear that there is a math expressions available

I'm not sure how these should be read out, but if you cannot see the expressions, the first one for example is `pi r squared`, I would expect that at the very least, NVDA would read "math expression" which would lead the user to investigate why they do not hear more and eventually download a component like [MathCAT](https://nsoiffer.github.io/MathCAT/)

### NVDA logs, crash dumps and other attachments:

N/A

#### NVDA version:

2024.4.2 (35031)

#### Windows version:

10

#### Name and version of other software in use when reproducing the issue:

Chrome (latest) and Firefox (latest)

#### Other information about your system:

N/A

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

Earlier versions also have the issue

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by nvdaes** [comment]
@NSoiffer , you may be interested in this. In case I mention you.

**Comment by NSoiffer** [comment]
The plan is to make MathCAT be a part of NVDA so there is math support out of the box. The plan is to do this so that one can still choose addons that support MathML as alternatives if MathCAT isn't your preferred solution.

**Comment by nvdaes** [comment]
Neil wrote:

> The plan is to make MathCAT be a part of NVDA so there is math support out of the box. 

I agree with you, and personally I wouldn't consume much time in this since MathCAT is functional and working, so I would prefer to prioritize the integration of MathCAT with NVDA as soon as possible. But I mentioned you in case your opinion was different, and to know it.

**Comment by G0dwin** [comment]
> The plan is to make MathCAT be a part of NVDA so there is math support out of the box

That would be ideal. Can you share a timeline or ETA?

**Comment by NSoiffer** [comment]
I really hoped to have it ready for 2025.1, but unless a miracle happens, it isn't going to be ready by then.

**Comment by seanbudd** [comment]
In the mean time it would be good to report something if no math handler is installed

**Comment by CyrilleB79** [comment]
@seanbudd, this issue has been closed with the inclusion of MathCAT in NVDA. But MathCAT does not work with Chrome UIA, and in this case, only the symbols are read. Given some synth do not read all symbols, we can end up with nothing reported at all when reaching a math expression. Even if with symbols that can be read by the synth, it would help to know that they are part of a math expression.

So I guess we should reopen this issue, although with a lower priority, until MathCAT can be used in any type of content.

**Comment by seanbudd** [comment]
I think it would be better to open a new issue that specifically covers the UIA case

**Comment by CyrilleB79** [comment]
OK. I have opened #19852.
